x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/search/WeightedTokensQueryBuilder.java

```diff
@@ -5,13 +5,8 @@
  * 2.0.
  */
 
-package org.elasticsearch.xpack.ml.queries;
+package org.elasticsearch.xpack.core.ml.search;
```

Comment:
I am curios why this `QueryBuilder` is in `xpack.core.ml.search` package?

Reply:
Where do you think would be a better place to put it?

Reply:
I was wondering why it is in the same package as other queries: `SparseVectorQueryBuilder` and `TextExpansionQueryBuilder`?  Is its place necessary for imports?

Reply:
Ah, thanks for the clarifying question. This is moved to core because it will be needed for subsequent `semantic_text` work.